stl/inc/format

```diff
@@ -191,6 +213,32 @@ constexpr const _CharT* _Parse_width(const _CharT* _Begin, const _CharT* _End, _
     return _Begin;
 }
 
+template <class _CharT, _Parse_spec_callbacks<_CharT> _Callbacks_type>
+constexpr const _CharT* _Parse_precision(const _CharT* _Begin, const _CharT* _End, _Callbacks_type&& _Callbacks) {

```

Comment:
Pre-existing, no change requested: We generally name parameters `_First` and `_Last` instead of `_Begin` and `_End`. This follows STL parameter naming conventions (begin/end refer to an entire container; first/last refers to the range passed to an algorithm), and makes more sense when we increment first etc.

Reply:
added todo card to project